### PR TITLE
R 2.1.0

### PR DESCRIPTION
**Pull request by phax**
Further minor code optimizations

### CODE REVIEW

File: modules/holodeckb2b-common/src/main/java/org/holodeckb2b/common/messagemodel/util/CompareUtils.java [hunk 1]
```diff
@@ -31,6 +31,10 @@
  * @author Sander Fieten <sander at holodeck-b2b.org>
  */
 public final class CompareUtils {
+  
+    private static <T> boolean _nullSafeEqual (final T o1, final T o2) {
```
Comment (sfieten) [inline modules/holodeckb2b-common/src/main/java/org/holodeckb2b/common/messagemodel/util/CompareUtils.java]: This method should have Javadoc

**Comment by phax** [comment]
The last change is important when using together with the E-SENS connector!

**Comment by sfieten** [comment]
Hi Philip, thanks already for this PR! 
I think we should make the change in the sortFiles method permanent and use Arrays.sort. Therefore I think we can remove the old version from the source. Would however prefer to have inline Comparator.
Can you make these changes?

**Comment by phax** [comment]
Et voila :)

**Comment by sfieten** [comment]
Thanks again Philip for your contributions!

**Pull request merged**